sections/basics/coming-from-css.js

```diff
@@ -60,6 +60,33 @@ component \`Counter\` and the Styled Component \`StyledCounter\`
 don't clash names but remain easily identifiable in the React Developer 
 Tools and Web Inspector.
 
+### Define Styled Components outside of the render method
+
+It is important to define your styled components outside of render method, 
+otherwise the styled component will be recreated on every single render pass.
+Defining a styled component within the render method is bad for performance
+and should be avoided.
+
+Write your styled components the recommend way:
+
+\`\`\`jsx
+const StyledWrapper = styled.div\`/* ... */\`
+
+const Wrapper = ({message}) => {
+  return <StyledWrapper>{message}</StyledWrapper>
+}
+\`\`\`
+
+Instead of:
+
+\`\`\`jsx
+const Wrapper = ({message}) => {
+  const StyledWrapper = styled.div\`/* ... */\`
```

Comment:
Let's add a comment above this line saying `// WARNING: THIS IS VERY VERY BAD AND SLOW, DO NOT DO THIS!!!` so that people don't accidentally copy this without reading the text and assume it's correct

Reply:
I added that comment.